bundles/framework/mydata/view/PublishedMaps/PublishedMapsList.jsx

```diff
@@ -49,81 +51,70 @@ export const PublishedMapsList = ({ controller, data = [], loading }) => {
             dataIndex: 'updated',
             sorter: getSorterFor('updated'),
+            width: 135,
             render: title => Oskari.util.formatDate(title)
         },
-        {
-            align: 'left',
-            title: <Message messageKey='tabs.publishedmaps.grid.domain' />,
-            dataIndex: 'pubDomain',
-            sorter: getSorterFor('pubDomain'),
-        },
-        {
-            align: 'left',
-            title: <Message messageKey='tabs.publishedmaps.grid.publish' />,
-            dataIndex: 'isPublic',
-            sorter: getSorterFor('isPublic'),
-            render: (title, item) => {
-                if (item.isPublic) {
-                    return (
-                        <Confirm
-                            title={<Message messageKey='tabs.publishedmaps.popup.unpublishmsg' messageArgs={{ name: item.name }} />}
-                            onConfirm={() => controller.setPublished(item)}
-                            okText={<Message messageKey='tabs.publishedmaps.button.ok' />}
-                            cancelText={<Message messageKey='tabs.publishedmaps.button.cancel' />}
-                            placement='bottomLeft'
-                        >
-                            <a><Message messageKey='tabs.publishedmaps.unpublish' /></a>
-                        </Confirm>
-                    )
-                } else {
-                    return (
-                        <a onClick={() => controller.setPublished(item)}><Message messageKey='tabs.publishedmaps.publish' /></a>
-                    )
-                }
-            }
-        },
-        {
-            align: 'left',
-            title: <Message messageKey='tabs.publishedmaps.grid.show' />,
-            render: (title, item) => {
-                return (
-                    <a onClick={() => controller.showOnMap(item)}><Message messageKey='tabs.publishedmaps.show' /></a>
-                );
-            }
-        },
-        {
-            align: 'left',
-            title: <Message messageKey='tabs.publishedmaps.grid.html' />,
-            dataIndex: 'id',
-            render: (title, item) => {
-                return (
-                    <a onClick={() => controller.showHtml(item)}><Message messageKey='tabs.publishedmaps.grid.html' /></a>
-                );
-            }
-        },
         {
             align: 'left',
             title: <Message messageKey='tabs.publishedmaps.grid.actions' />,
             dataIndex: 'id',
-            width: 100,
+            width: 150,
             render: (title, item) => {
                 return (
                     <ToolsContainer>
+                        <Tooltip title={<Message messageKey='tabs.publishedmaps.show' />}>
+                            <div className='icon t_show' onClick={() => controller.showOnMap(item)}>
+                                <PictureOutlined style={ ICON_STYLE } />
+                            </div>
```

Comment:
Let's try wrapping these in AntD buttons for accessibility (might need to style them a bit more to make them look good). Also changed  `icon` classname to `t_icon` for consistency:
```suggestion
<Button className='t_icon t_show' icon={<PictureOutlined style={ ICON_STYLE } />} onClick={() => controller.showOnMap(item)}/>
```